templates/web/fixmystreet.com/report/new/roads_message.html

```diff
@@ -29,6 +29,9 @@
             <a href="https://publicrightsofway.oxfordshire.gov.uk/web/standardmap.aspx">this page</a>.
         </p>
     </div>
+    <div id="js-not-council-car-park" class="hidden js-responsibility-message">
+        <p>There is no Buckinghamshire maintained car park in the area selected.</p>
```

Comment:
Perhaps could be `<p>There is no council car park in the area selected.</p>` and then if other councils do the same the message works?